Frontend/src/components/ActiveCripto.jsx

```diff
@@ -0,0 +1,33 @@
+import React,{useEffect,useState} from 'react'
+import {Cripto} from './Cripto'
+
+import UserService from "../services/user.service";
+
+const ActiveCripto = () => {    
+    const [currentCriptos, setCurrentCriptos]= useState([])
+    
+    useEffect(() => {
+        const fetchData = async () => {
+            const cripto = await UserService.getActiveCriptos()
+            console.log(cripto)
+            setCurrentCriptos(cripto.data)                             
+        }
+        fetchData()
+    }, [setCurrentCriptos])
+
+    return (
+        <div className='criptos-container'>
+            {currentCriptos.map( cripto =>
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/JeremiasFuentes/unq-desapp-fuentes-diaz/pullRequest?prid=8198099)
```suggestion
            {currentCriptos.map( (cripto) =>
```